### ISSUE TITLE
Change in customized styles and code snippets

### ISSUE THREAD

**Issue by levinjb** [piroor/treestyletab#2404]
## Short description
A customization of tab labels suddenly stopped working.
This is a problem with back compatibility or out-of-date documentation.
No fix to the code is required.
(Also note - I am a user, _not_ a developer, though I am a software engineer with some understanding of CSS)

## Steps to reproduce
Please excuse the form of this report, as it is not really a bug feature problem, as it turns out.
I had a style customization in Options / Advanced / Extra Style Rules box:

```
.tab.unread .label {
  color: #ee0066 !important;
  font-style: italic !important;
}

```
I noticed that it stopped working; I believe the time corresponded to the release of TST 3.2.0 or 3.2.1; I had upgraded to FireFox 70 a week earlier, and there is a small chance that the problem originated there.

## Expected result

Red label in a new unread tab

## Actual result

The label was displayed in black. (When I changed the "too long label" option from Fade-out to Crop, the label remained black but the ".." was displayed in red.)

Upon examination of some code snippets on GitHub I noticed that where I had "label" the snippets had "label-content". Changing my style rules to use "label-content" caused the changes I had intended (and previously implemented with "label") to work. (Although now the ".." is black.)

I do not know when "label" alone stopped working and "label-content" became required for this functionality. I am just reporting that this is an issue which may affect other users who like to customize their TST appearance. It is my hope that this information may be of some use.

## Environment

 * Platform (OS): Windows 7
 * Version of Firefox: 70.0 (and 70.0.1)
 * Version (or revision) of Tree Style Tab: 3.2.1

**Comment by namitjuneja** [comment]
For me as well 

```
.tab .active .label{
color: black;
} 
```
stopped wokring

**Comment by levinjb** [comment]
As I said above, this can be fixed by changing the ".label" to ".label-content". I would call this neither a fix nor a workaround, but a change in the CSS code needed to effect the desired style modification. But in any case, you can make it work.

**Comment by piroor** [comment]
I thinks the change 3e15ade9dfe96c6e5395cb145a361c6e4f594846 should improves backward compatibility for CSS rules used in old versions. Could you try the latest development build?

**Comment by levinjb** [comment]
> I thinks the change [3e15ade] should improves backward compatibility for CSS rules used in old versions. Could you try the latest development build?

As I said before, I'm a user, not a developer. I need to know how to find and install the development build, or I'll wait and you can let me know when this change is included in a release. I looked at the change, but I'm not sure how it would affect the issue I had, having to change ".label" to ".label-content" -- but I'm no CSS expert.

**Comment by levinjb** [comment]
Your reply on #2401 says "It is a natural result based on the CSS spec. You need to use .label-content instead of .label" -- but something changed, because .label used to work.

**Comment by piroor** [comment]
Here is instructions to try the development build: https://github.com/piroor/treestyletab/blob/master/README.md#development-builds

**Comment by levinjb** [comment]
I downloaded and temporarily installed TST 3.2.1.9687 per your directions.  Then I returned to the TST configuration and changed the .label-content element back to .label element (in ".tab.unread .label {", "tab-item.active .label {", and "tab-item.discarded .label {") and everything worked the way it had previously. I conclude that change 3e15ade has fixed the problem I saw.  Thank you.

**Comment by piroor** [comment]
After more research I've got completely what caused the problem. It is clearly a regression by my mistake. I missed that a CSS selector `.tab .label-container` was replaced to `tab-item tab-label-container` unexpectedly and the replacement deactivated a style definition for backward compatibility exact same to 3e15ade. Thus I've removed duplicated workaround with the commit 1b0b18508. It's very shameful...

**Comment by levinjb** [comment]
Well, don't be embarrassed - This kind of thing happens often enough in long-lived code, as I know from experience; I'm glad it is cleared up. Thanks for taking the effort for such a relatively small and unimportant problem.   --/JBL

**Comment by levinjb** [comment]
And finally, TST release 3.2.2 on Nov 4 has fixed the problem and returned functionality to what it was before 3.2.0. This should not be labeled "fixed".
